### ISSUE TITLE
Avis home page! End Rent Date picker screen reader does not announce the date we havve selected

### ISSUE THREAD

**Issue by nsingh2001** [Kishan1601/Test-1#7]
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5
Repro Steps
1.	Open the URL https://www.avis.com/en/home

2.Press Tab key to navigate to the date picker select a date 
3.check if the screen reader announce the selected date
Expected Result
Change of date in date picker using arrow keys should be announce by the screen reader
Actual Result
In begin date picker user is unable to navigate and select a date in calendar using arow keys as the screen reader does not announce the change of date using arrow keys
WCAG Guideline
https://www.w3.org/TR/WCAG21/#keyboard
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5
Repro Steps
1.	Open the URL https://www.avis.com/en/home

2.Press Tab key to navigate End Rent Date picker to the date picker select a date 
3.check if the screen reader announce the selected date
Expected Result
Change of date in date picker using arrow keys should be announce by the screen reader
Actual Result
In end date picker user is unable to navigate and select a date in calendar using a row keys as the screen reader does not announce the change of date using arrow keys
WCAG Guideline
https://www.w3.org/TR/WCAG21/#keyboard
